aws_etl_tools/redshift_database.py

```diff
@@ -38,6 +40,8 @@ def _compose_unload_query(self, query, s3_path, options):
         query_commands.append('ADDQUOTES') if options.get('add_quotes', False) else None
         query_commands.append('ESCAPE') if options.get('escape', False) else None
         query_commands.append('HEADER') if options.get('header', False) else None
+        query_commands.append('%s' % options.get('compression_type')) if options.get('compression_type', False) else None
```

Comment:
maybe `str(options.get('compression_type'))` instead of the string interpolation?